tilgangskontroll-tiltaksansvarlig/src/main/kotlin/no/nav/amt/tiltak/tilgangskontroll_tiltaksansvarlig/TiltaksansvarligAutoriseringServiceImpl.kt

```diff
@@ -59,4 +59,10 @@ class TiltaksansvarligAutoriseringServiceImpl(
 		}
 	}
 
+	override fun harTilgangTilSkjermedePersoner(navAnsattAzureId: UUID) : Boolean {
+		return adGruppeService.hentAdGrupper(navAnsattAzureId)
+			.any { it.name == AdGrupper.EGEN_ANSATT_GRUPPE }
+
+	}
```

Comment:
Vet at vi har gjort det slik for `TILTAKSANSVARLIG_ENDRINGSMELDING_GRUPPE` men dette er egentlig ikke greit å gjøre. Vi må gå over til å bruke IDen til gruppen istedenfor. Når det gjelder skjermet person så er det nok best å hente svaret fra poao-tilgang

Reply:
Er det riktig nå?

Reply:
Er det forskjellig id i dev og prod?